wazimap/static/js/profile_map.js

```diff
@@ -152,7 +152,7 @@ var ProfileMaps = function() {
                     layer.setStyle(self.layerStyle);
                 });
                 layer.on('click', function() {
-                    window.location = '/profiles/' + feature.properties.level + '-' + feature.properties.code + '/';
+                    window.location = '/profiles/' + feature.properties.level + '/' + feature.properties.code + '/' + feature.properties.name;
```

Comment:
Why is this been changed?

Reply:
I've updated the paths to the recommended way of structuring them in Django 2 and what the industry standard currently is.